contentmgr/staging.go

```diff
@@ -232,8 +232,8 @@ func (cm *ContentManager) processStagingZone(ctx context.Context, zone util.Cont
 	ctx, span := cm.tracer.Start(ctx, "aggregateContent")
 	defer span.End()
 
-	grpLocs, err := cm.getStagedContentsGroupedByLocation(ctx, zone.ID)
-	if err != nil {
+	var grpLocs []string
+	if err := cm.db.Model(&util.Content{}).Where("active and aggregated_in = ?", zone.ID).Distinct().Pluck("location", &grpLocs).Error; err != nil {
```

Comment:
please check the length here before proceeding

```
if len(out) == 0 {
		return nil, fmt.Errorf("no location for staged contents")
	}

Reply:
Done